addons/crm/views/crm_lead_views.xml

```diff
@@ -791,7 +791,7 @@
                         options="{'currency_field': 'company_currency'}" groups="crm.group_use_recurring_revenues"/>
                     <field name="recurring_plan" optional="hide" groups="crm.group_use_recurring_revenues"/>
                     <field name="stage_id_color" column_invisible="1"/>
-                    <field name="stage_id" optional="show" widget="badge_rotting" options="{'color_field': 'stage_id_color'}"/>
+                    <field name="stage_id" optional="show" widget="badge_rotting" options="{'no_open': True}"/>
```

Comment:
I added the `no_open` option to keep the behaviour we got when we used `selection_badge` widget inside `badge_rotting` widget (that is, with the selection_badge widget it was not possible to go to the form view of stage).